layers/meta-balena-raspberrypi/conf/machine/revpi-connect-4.conf

```
@@ -0,0 +1,11 @@
+#@TYPE: Machine
+#@NAME: Revolution Pi Connect 4
+#@DESCRIPTION: Machine configuration for Revolution Pi Connect 4 board
+
+require conf/machine/include/revpi4.inc
+
+KERNEL_DEVICETREE:append = " \
+	overlays/revpi-connect.dtbo \
+	overlays/revpi-connect-dt-blob-overlay.dtb \
```

```suggestion
	overlays/revpi-dt-blob-overlay.dtb \
```

This dt-blob contains everything needed for hat eep